src/content/docs/en/reference/api-reference.mdx

```diff
@@ -242,6 +252,14 @@ Invalidates a cookie by setting the expiration date in the past (0 in Unix time)
 
 Once a cookie is "deleted" (expired), `Astro.cookies.has()` will return `false` and `Astro.cookies.get()` will return an [`AstroCookie`](#astrocookie) with a `value` of `undefined`. Options available when deleting a cookie are: `domain`, `path`, `httpOnly`, `sameSite`, and `secure`.
 
+##### `merge`
+
+<p>
+**Type:** `(cookies: AstroCookies) => void`
+</p>
+
+Merges a new `AstroCookies` instance into the current instance. Any new cookies will be added to the current instance, overwriting any existing cookies with the same name.
```

Comment:
I think this probably needs an “and” to separate the two clauses as it is not the _new_ cookies that overwrite, but only the ones matching existing cookies.

```suggestion
Merges a new `AstroCookies` instance into the current instance. Any new cookies will be added to the current instance and any cookies with the same name will overwrite existing values.
```